app/assets/stylesheets/image_x.scss

```diff
@@ -71,3 +77,21 @@ $image-thumb-slider-border-color: #000;
 .#{$namespace}-image-x-label {
   text-align: center;
 }
+
+.frame {
+    position:absolute;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, display, position<br>Colon after property should be followed by one space